### PR TITLE
Add tomas's blog post (part i)

### PR DESCRIPTION
**Pull request by m52go**
Blog post by @KanoczTomas. I think it's ready to go.

### CODE REVIEW

**Comment by KanoczTomas** [comment]
@m52go this part of the commit looks like a left over from a previous post maybe?
`
excerpt: "The Bisq DAO enables value transfer and decision-making for the Bisq network. But it has several additional benefits too, which we'll focus on in this series.<br><br>In this post, Part I of this series, we'll discuss how Bisq's revenue model offers a new approach to funding open-source software development that, by design, avoids incentives to exploit user privacy altogether.<br><br>"
`

**Comment by m52go** [comment]
@KanoczTomas you're right, thanks for catching that. I forgot to change. Fixed. 

Let me know if you prefer a different excerpt.

File: _posts/2019-03-22-how-to-set-up-bitcoin-regtest.md [hunk 1]
```diff
@@ -0,0 +1,119 @@
+---
+layout: post
+title: "How to Set Up a Bitcoin Regtest Environment"
+author: Tomáš Kanócz
+excerpt: "The Bisq DAO enables value transfer and decision-making for the Bisq network. But it has several additional benefits too, which we'll focus on in this series.<br><br>In this post, Part I of this series, we'll discuss how Bisq's revenue model offers a new approach to funding open-source software development that, by design, avoids incentives to exploit user privacy altogether.<br><br>"
+---
```
Comment (ManfredKarrer) [inline _posts/2019-03-22-how-to-set-up-bitcoin-regtest.md:6]: excerpt is from prev. post

**Review by ManfredKarrer — CHANGES_REQUESTED** [review]

File: _posts/2019-03-22-how-to-set-up-bitcoin-regtest.md (hunk 1 above)
Comment (ManfredKarrer) [reply _posts/2019-03-22-how-to-set-up-bitcoin-regtest.md:6]: Beside the excerpt utACK

Comment (m52go) [reply _posts/2019-03-22-how-to-set-up-bitcoin-regtest.md:6]: Thanks, latest commit fixed that.

**Comment by m52go** [comment]
@ManfredKarrer excerpt fixed in latest commit, would you mind formally accepting? I know you utACKed above but GitHub still shows "Changes requested"

**Comment by KanoczTomas** [comment]
@m52go it still shows the old excerpt when I look at the latest commit.

**Comment by KanoczTomas** [comment]
@m52go ah sorry, was looking at the wrong commit :) Text is fine, thank you! @ManfredKarrer it can go out.

**Review by ManfredKarrer — APPROVED** [review]
utACK

**Pull request merged**